js/source/geojson_source.js

```diff
@@ -1,28 +1,23 @@
 'use strict';
 
-var util = require('../util/util');
 var Evented = require('../util/evented');
-var TilePyramid = require('./tile_pyramid');
-var Source = require('./source');
+var util = require('../util/util');
 var urlResolve = require('resolve-url');
 var EXTENT = require('../data/bucket').EXTENT;
 
-module.exports = GeoJSONSource;
+module.exports.create = function (id, options, dispatcher) {
+    return new GeoJSONSource(id, options, dispatcher);
+};
```

Comment:
Why export a method that calls the constructor rather than export the constructor, as before?